docs/_data/paths/rating-proposals_forecasts.yaml

```diff
@@ -146,8 +146,9 @@ patch:
       application/vnd.trolie.rating-forecast-proposal.v1+json:
         schema:
           $ref: ../components/schemas/forecast-rating-proposal.yaml
-        example:
-          $ref: ../examples/forecast-ratings-proposal-patch.json
+        examples:
+          simple-example:
+            externalValue: '/example-json/forecast-ratings-proposal-patch.json'
```

Comment:
I like this best, but there was at least one tool (an examples validator) which would not follow this reference. We're not going to use that tool--Redocly validates examples and generates the doc viewer just fine--but we might want to use a few code generators to make sure they don't choke.

We can use `examples` with inline or `externalValue`, of course.